src/config-types.ts

```diff
@@ -74,5 +74,6 @@ export type Options = {
   };
   misc: {
     prefetch_links: 'in-viewport' | 'off';
+    sequential_compress: boolean;
```

Comment:
Maybe I would have made a cli option for this because it's not configuring the optimisation but altering the technical aspects of how it's apply.
Not a big deal.

Reply:
Can definitely do